features/lima/README.md

```diff
@@ -4,17 +4,30 @@
 This feature flag produces an image suitable for using with [lima](https://lima-vm.io)
 </website-feature>
 
-Garden Linux images for Lima are published at https://images.gardenlinux.io
+Garden Linux images for Lima are published with nightly or release
 
 How to use the pre-built image:
 
-1. Create and start the VM
+1. Prerequisities
+    - We use glrd to get image link, this needs 
+        - sudo access
+        - podman
+        - access to gardenlinux-glrd.s3.eu-central-1.amazonaws.com
 
+2. Create and start the VM
+
+Lima image can be started with start-lima.py inside bin folder
+```
+# for the latest nightly image, use:
+python3 bin/start-lima.py --name gardenlinux --version nightly --arch <arch>
```

Comment:
I would remove the `python3` call and maybe reference what is supported by arch or mention the `--help` flag.